docs/cloud/http-api.md

```diff
@@ -182,6 +184,28 @@ So if the delete query matches a branch, **every** series under that branch will
 curl -u "api_key:<Your Grafana.com API Key>" https://<tsdbgw>/metrics/delete -d query=some.series.to.delete.*
 ```
 
+#### Tagged With `/tags/delSeries`
+
+Deletes metrics which match the `path` parameter(s).
+Note that the data stays in the datastore until it expires.
+Should the metrics enter the system again with the same metadata, the data will show up again.
+
+
+* Method: POST
+* API key type: Editor
+
+##### Parameters
+
+* user name: `api_key`
+* password: Your Grafana.com API Key
+* path (required, multiple allowed): A single Graphite series
+
+##### Example
+
+```sh
+curl -u "api_key:<Your Grafana.com API Key>" https://<tsdbgw>/tags/delSeries -d path=some.series;key=value -d path=another.series;tag=value
```

Comment:
same here, needs to be quoted

Reply:
blah, 455f836e6f1c15d9c8d9a29563a8d075b1dc4d09